proc_fieldSweep.py

```diff
@@ -0,0 +1,61 @@
+from pylab import *
```

Comment:
not sure why this is in this PR, I tried diffing against master but nothing came up so I think that means there is no difference but then why is it showing up here?

Reply:
this is not in master

Reply:
this should not be added to the root directory. move to examples?  this does seem very minimalist, which is good.